crates/gitbutler-branch-actions/src/status.rs

```diff
@@ -98,18 +92,10 @@ pub fn get_applied_status_cached(
     let vb_state = ctx.project().virtual_branches();
     let default_target = vb_state.get_default_target()?;
 
-    let locks = if ctx.project().use_experimental_locking {
-        compute_locks(
-            ctx,
-            &workspace_head,
-            &default_target.sha,
-            &base_diffs,
-            &virtual_branches,
-        )?
-    } else {
-        let base_tree = ctx.repository().find_commit(default_target.sha)?.tree()?;
-        compute_old_locks(ctx.repository(), &base_diffs, &virtual_branches, base_tree)?
-    };
+    let workspace_dependencies =
+        compute_workspace_dependencies(ctx, &default_target.sha, &base_diffs, &virtual_branches)?;
```

Comment:
@krlvi  I think we discussed a thing related to this line earlier today, would you prefer ok or default here as to not give the user a "something went wrong"?